app/const.py

```diff
@@ -4,3 +4,130 @@
+    "newcastle-staffs.gov.uk": [
+        "Newcastle-Under-Lyme Town Centre",
+        "Kidsgrove",
+        "Newcastle-under-Lyme",
+        "Newcastle-under-Lyme Town Centre",
+    ],
+    "northdevon.gov.uk": ["Barnstaple"],
+    "ne-derbyshire.gov.uk": ["Clay Cross"],
+    "nelincs.gov.uk": ["Grimsby Town Centre", "Grimsby"],
+    "northlincs.gov.uk": ["Scunthorpe"],
+    "northumberland.gov.uk": ["Blyth Town Centre", "Blyth"],
+    "northyorks.gov.uk": ["Northallerton", "Scarborough", "Whitby"],
+    "norwich.gov.uk": ["Norwich"],
+    "nottinghamcity.gov.uk": ["Nottingham City Centre, West End Point"],
+    "oldham.gov.uk": ["Oldham"],
+    "pendle.gov.uk": ["Nelson"],
+    "peterborough.gov.uk": ["Peterborough"],
+    "plymouth.gov.uk": ["Plymouth City Centre"],
+    "portsmouth.gov.uk": ["Fratton", "Commercial Road"],
+    "preston.gov.uk": ["Preston"],
+    "redcar-cleveland.gov.uk": ["Redcar", "Loftus"],
+    "redditchbc.gov.uk": ["Redditch"],
+    "rochdale.gov.uk": ["Rochdale Town Centre", "Rochdale"],
+    "rotherham.gov.uk": ["Rotherham"],
+    "sandwell.gov.uk": ["West Bromwich", "Rowley Regis", "Smethwick"],
+    "sefton.gov.uk": ["Southport"],
+    "sheffield.gov.uk": ["Sheffield High Street", "Stocksbridge"],
+    "somerset.gov.uk": ["Yeovil", "Bridgwater", "Glastonbury", "Taunton"],
+    "southglos.gov.uk": ["Kingswood"],
+    "southkesteven.gov.uk": ["Grantham"],
+    "southribble.gov.uk": ["Leyland"],
+    "southtyneside.gov.uk": ["South Shields"],
+    "southwark.gov.uk": ["Old Kent Road"],
+    "staffordbc.gov.uk": ["Stafford"],
+    "stevenage.gov.uk": ["Stevenage"],
+    "sthelens.gov.uk": ["St Helens"],
+    "stockport.gov.uk": ["Stockport", "Cheadle"],
+    "stockton.gov.uk": ["Stockton", "Thornaby"],
+    "sunderland.gov.uk": ["Sunderland City Centre"],
+    "sutton.gov.uk": ["Sutton High Street", "Sutton"],
+    "swindon.gov.uk": ["Swindon"],
+    "tamworth.gov.uk": ["Tamworth Town Centre"],
+    "teignbridge.gov.uk": ["Newton Abbot"],
+    "telford.gov.uk": ["Telford"],
+    "thanet.gov.uk": ["Ramsgate", "Margate"],
+    "thurrock.gov.uk": ["Tilbury", "Grays"],
+    "torbay.gov.uk": ["Paignton", "Torquay"],
+    "trafford.gov.uk": ["Stretford"],
+    "wakefield.gov.uk": ["Wakefield", "Castleford"],
+    "walsall.gov.uk": ["Walsall", "Bloxwich"],
+    "wandsworth.gov.uk": ["Putney Town Centre"],
+    "warrington.gov.uk": ["Warrington"],
+    "warwickdc.gov.uk": ["Leamington Town Centre"],
+    "wigan.gov.uk": ["Wigan"],
+    "wiltshire.gov.uk": ["Trowbridge", "Salisbury City Centre"],
+    "wirral.gov.uk": ["New Ferry", "Birkenhead"],
+    "wolverhampton.gov.uk": ["Wolverhampton", "Wolverhampton City Centre (West)"],
+    "worcester.gov.uk": ["Blackfriars - Northern City Centre", "Worcester"],
+    "wyreforestdc.gov.uk": ["Kidderminster"],
+    "bcpcouncil.gov.uk": ["Bournemouth"],
+    "eastsuffolk.gov.uk": ["Lowestoft"],
+    "northnorthants.gov.uk": ["Corby"],
+    "westnorthants.gov.uk": ["Northampton"],
+}
```

Comment:
Could these lists be converted to tuples as they're a more efficient data structure to use if we don't care about their immutability

Reply:
Yes, they're now tuples